README.md

```diff
@@ -478,6 +478,13 @@ These are the available config options for making requests. Only the `url` is re
     
     // throw ETIMEDOUT error instead of generic ECONNABORTED on request timeouts
     clarifyTimeoutError: false,
+  },
+
+  // `defaultReject`, when set to a function, is executed when your axios request is rejected
+  // and you haven't explicitly used a catch block
+  // it implements the unhandledRejection event
+  defaultReject: function (err) {
```

Comment:
would also add `defaultResolve`

Reply:
I don't know how to listen to unhandled promise resolutions